weblate/templates/snippets/info.html

```diff
@@ -50,6 +50,15 @@ <h4 class="panel-title">{% translate "Overview" %}</h4>
                 </td>
               </tr>
             {% endif %}
+
+            {% if project.all_reviewers %}
+              <tr>
+                <th>{% translate "Project Reviewers" %}</th>
```

Comment:
Label consistency with others:

```suggestion
                <th>{% translate "Project reviewers" %}</th>
```